app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -87,6 +92,14 @@ def zones
     end
     alias_method :available_zones, :zones
 
+    def root_device_from_ami(ami_id)
+      ami_desc = client.describe_images('image-id' => ami_id)
+      root_device_name = ami_desc.body["imagesSet"][0]["rootDeviceName"]
+      root_device_definition = ami_desc.body["imagesSet"][0]["blockDeviceMapping"].find {|x| x["deviceName"] == root_device_name}
+      root_device_size = root_device_definition["volumeSize"]
+      return {:DeviceName => root_device_name, 'Ebs.VolumeSize' => root_device_size}
```

Comment:
In Ruby, method will return the return value of the last evaluated statement so no need to add `return` keyword.